### PR TITLE
Rcas 1.1.1

### PR DESCRIPTION
**Pull request by dyusuf**

### CODE REVIEW

File: tools/rna_tools/rcas/.shed.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: rcas
+owner: rnateam
+description: RNA Centric Annotation System that provides intuitive reports and publication ready graphics.
+homepage_url: git@github.com:BIMSBbioinfo/RCAS.git
```
Comment (bgruening) [inline tools/rna_tools/rcas/.shed.yml]: This should be a http URL :)

File: tools/rna_tools/rcas/rcas.xml [hunk 2]
```diff
@@ -0,0 +1,306 @@
+<tool id="rcas" name="RNA Centric Annotation System" version="1.1.1">
+    <requirements>
+            <requirement type="package" version="1.1.1">bioconductor-rcas</requirement>
```
Comment (bgruening) [inline tools/rna_tools/rcas/rcas.xml]: indentation?

File: tools/rna_tools/rcas/rcas.xml [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+    </requirements>
+    <stdio>
+        <regex match="Execution halted"
+           source="both"
+           level="fatal"
+           description="Execution halted." />
+        <exit_code range="1:" />
+    </stdio>
+    <command><![CDATA[
+            Rscript $script_file
```
Comment (bgruening) [inline tools/rna_tools/rcas/rcas.xml]: single-quotes

File: tools/rna_tools/rcas/rcas.xml [hunk 4]
```diff
@@ -0,0 +1,306 @@
...
+    <inputs>
+        <param name="genomeVersion" type="select" label="Genome Version">
+            <option value="hg19" selected="true">hg19</option>
+            <option value="dm3">dm3</option>
+            <option value="ce10">ce10</option>
+            <option value="mm9">mm9</option>
+        </param>
+
+        <param name="input_BED" type="data" format="BED" label="Target regions in BED format"
+        help="The coordinates should be in accord with Genome Version."/>
```
Comment (bgruening) [inline tools/rna_tools/rcas/rcas.xml]: indentaion

File: tools/rna_tools/rcas/rcas.xml [hunk 5]
```diff
@@ -0,0 +1,306 @@
...
+
+        <param name="input_GTF" type="data" format="GTF"
+            label="Reference annotation in ENSEMBL GTF format"/>
+
+        <param name="run_anot" label="Run annotation."
+        type="boolean" falsevalue="FALSE" truevalue="TRUE" checked="True"
+        help="Use this option if you want to run annotation."/>
```
Comment (bgruening) [inline tools/rna_tools/rcas/rcas.xml]: indentation

File: tools/rna_tools/rcas/.shed.yml (hunk 1 above)
Comment (dyusuf) [inline tools/rna_tools/rcas/.shed.yml]: yes! what I was thinking.

File: tools/rna_tools/rcas/rcas.xml (hunk 3 above)
Comment (dyusuf) [inline tools/rna_tools/rcas/rcas.xml]: even for the config file?

File: tools/rna_tools/rcas/rcas.xml (hunk 2 above)
Comment (dyusuf) [inline tools/rna_tools/rcas/rcas.xml]: I don't understand.

**Comment by dyusuf** [comment]
@bgruening the travis output shows the error for GraphClust. It seems not related to RCAS.

File: tools/rna_tools/rcas/rcas.xml (hunk 3 above)
Comment (bgruening) [inline tools/rna_tools/rcas/rcas.xml]: The configfile is stored by Galaxy in some temp path. And this can container whitespaces or whatever. So yes :)

File: tools/rna_tools/rcas/rcas.xml (hunk 2 above)
Comment (bgruening) [inline tools/rna_tools/rcas/rcas.xml]: I think the indentation is wrong, 8 spaces and not 4?

**Comment by bgruening** [comment]
@dyusuf strange can you rebase your branch against master?
You error is here and it seems your test data is not up to date or it is random: https://travis-ci.org/bgruening/galaxytools/jobs/220564929#L2192

**Comment by bgruening** [comment]
Test data is still not valid :(

**Comment by dyusuf** [comment]
yeah... let me see if i can fix it

File: tools/uniprot_rest_interface/uniprot.xml [hunk 6]
```diff
@@ -6,14 +6,8 @@
     <requirements>
         <requirement type="package" version="2.7">requests</requirement>
     </requirements>
```
Comment (bgruening) [inline tools/uniprot_rest_interface/uniprot.xml]: @dyusuf here is still something wired in this PR.

File: tools/uniprot_rest_interface/test-data/test2_retrieve.gff [hunk 7]
```diff
@@ -1,16 +1,7 @@
 ##gff-version 3
 ##sequence-region S0DS17 1 369
-S0DS17	UniProtKB	Chain	1	369	.	.	.	ID=PRO_0000437163;Note=Cytochrome P450 momooxygenase apf8
```
Comment (bgruening) [inline tools/uniprot_rest_interface/test-data/test2_retrieve.gff]: These files should not be touched.

Comment (dyusuf) [inline tools/uniprot_rest_interface/test-data/test2_retrieve.gff]: Hmm... I guess these unrelated changes are because of rebasing against the master few days ago. Check out the commit history after that. Probably, the changes in PR #548 got caught and become  part of this PR. What is the best practice this moment?

Comment (bgruening) [inline tools/uniprot_rest_interface/test-data/test2_retrieve.gff]: Mh, another rebase? You can also merge master into this branch or simply close it and create a new PR. Thanks Dili!
